src/pocketdb/repositories/web/WebRpcRepository.cpp

```diff
@@ -2132,6 +2132,85 @@ namespace PocketDb
         return result;
     }
 
+    UniValue WebRpcRepository::GetAccountEarning(const string& address, int height, int depth)
+    {
+        UniValue result(UniValue::VARR);
+
+        string sql = R"sql(
+            select
+                a.address,
+                ifnull(t.Type,0) as type,
+                sum(ifnull(o.Value, 0)) as amount
+            from (select ? address)a
```

Comment:
This code might lead to the sql injection as an external parameter is used here.
for instance if address is equal to `sqlte_version(); select 12345` the expression in brackets might become a form
```
(select sqlite_version(); select 12345 address)
```

Reply:
Also we are using `sqlite3_bind_text` to bind the value and at this point it is not possible to perfrom an sql injection